upstream-tools/replacements.json

```diff
@@ -19034,6 +19070,20 @@
       "new": "Using `pulumi import`, import Transfer Workflows using the  `id`. For example:"
     }
   ],
+  "website/docs/r/verifiedaccess_instance_trust_provider_attachment.html.markdown": [
+    {
+      "old": "% terraform import aws_verifiedaccess_instance_trust_provider_attachment.example vai-1234567890abcdef0/vatp-8012925589",
+      "new": "% TODO import aws_verifiedaccess_instance_trust_provider_attachment.example vai-1234567890abcdef0/vatp-8012925589"
+    },
+    {
+      "old": "In Terraform v1.5.0 and later, use an `import` block to import Verified Access Instance Trust Provider Attachments using the `verifiedaccess_instance_id` and `verifiedaccess_trust_provider_id` separated by a forward slash (`/`). For example:",
+      "new": "In TODO v1.5.0 and later, use an `import` block to import Verified Access Instance Trust Provider Attachments using the `verifiedaccess_instance_id` and `verifiedaccess_trust_provider_id` separated by a forward slash (`/`). For example:"
+    },
+    {
+      "old": "Using `terraform import`, import Verified Access Instance Trust Provider Attachments using the `verifiedaccess_instance_id` and `verifiedaccess_trust_provider_id` separated by a forward slash (`/`). For example:",
+      "new": "Using `TODO import`, import Verified Access Instance Trust Provider Attachments using the `verifiedaccess_instance_id` and `verifiedaccess_trust_provider_id` separated by a forward slash (`/`). For example:"
```

Comment:
Good catch. `TODO` is not descriptive, and should be updated to something like `Pulumi`. Will address in a follow-up commit to this PR soon.

Reply:
Hmm, looks like we have 608 TODOs within this file, spanning back to many months ago. WDYT if we submit this PR as is, and I'll create a follow-up PR to directly handle these TODOs before cutting a new release?

Reply:
Err. Yea could you backlog an item to clean this up? When doing that instead of cleaning up by hand we could improve the tooling not to require this. One place to check would be looking at registry docs for a resource with the TODO, if TODO renders in registry that's a clear indicator this is user-facing in a way.